frontend/src/pages/restartPage/moveAwayFromApDialog/MoveAwayFromApDialog.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import Dialog from "../../../components/atoms/dialog/Dialog";
+import Button from "../../../components/atoms/button/Button";
+
+import styles from "./MoveAwayFromApDialog.module.css";
+
+export type Props = {
+  active: boolean;
+  piTopIpAddress: string;
+  onSkip: () => void;
+};
+
+const getMessage = () => {
+  return (
+    <>
+      <span className={styles.dialogTitle}>Reconnect to your home Wi-Fi</span>
```

Comment:
I think we should refer to this as 'your main wifi' or just 'your wifi', not 'home wifi'. even though this process is designed for home users primarily, that is a unnecessarily specific assumption to make.